### PR TITLE
Sprint6 kali 

### PR DESCRIPTION
**Pull request by EthanIsakson**
Sprint 6 branch, includes:

- Hashcat Tutorial
--DockerFile
--Documentation
--Hashed Password File, Rockyou.txt wordlist.
- Hash-Identifier Tutorial
--DockerFile
--Documentation
--Hashed Password Files

- Necessary Schema per Logan's examples for Strings Tutorial.

### CODE REVIEW

**Comment by EthanIsakson** [comment]
Closes following issues: 
Closes #161
Closes #162

File: challenges/kali/challenges/hash-identifier/template.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+spec:
+  pods:
+    - metadata:
+        name: penimage
+        labels:
+          app: penimage
+      spec:
+        containers:
+          - name: penimage
+            image: ghcr.io/acasi-ctf/challenges/kali/penimage:latest
```
Comment (lgorence) [inline challenges/kali/challenges/hash-identifier/template.yaml:14]: This image doesn't exist.

File: challenges/kali/challenges/hashcat/template.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+spec:
+  pods:
+    - metadata:
+        name: penimage
+        labels:
+          app: penimage
+      spec:
+        containers:
+          - name: penimage
+            image: ghcr.io/acasi-ctf/challenges/kali/penimage:latest
```
Comment (lgorence) [inline challenges/kali/challenges/hashcat/template.yaml:14]: This image doesn't exist.

**Review by lgorence — CHANGES_REQUESTED** [review]
Looks like you are referencing the incorrect images in both template.yaml files and haven't pushed the Docker images you have Dockerfiles for.

File: challenges/kali/challenges/hashcat/template.yaml (hunk 2 above)
Comment (EthanIsakson) [reply challenges/kali/challenges/hashcat/template.yaml:14]: Oh stupid, I put that as a placeholder so I wouldn't forget.... haha and here we are

**Comment by EthanIsakson** [comment]
Updating paths to remove Kali to avoid confusion:
Closes #170

**Review by lgorence — APPROVED** [review]
Think everything looks good! 😄

**Pull request merged**